src/main/java/reposense/dataobject/RepoConfiguration.java

```diff
@@ -9,7 +9,6 @@
 
 import reposense.util.FileUtil;
 
-
```

Comment:
avoid unnecessary change if you didn't modify the file.

Reply:
Whoops, modified this file earlier but removed the changes xD